core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -277,6 +284,28 @@ private int maybeResetInputAndSeekToEndIntermediateTopicOffsets(final Map consum
             }
         }
 
+        if (!notFoundInputTopics.isEmpty()) {
+            System.out.println("Following input topics are not found, skipping them");
+            for (final String topic : notFoundInputTopics) {
+                System.out.println("Topic: " + topic);
+            }
+            topicNotFound = EXIT_CODE_ERROR;
+        }
+
+        if (!notFoundIntermediateTopics.isEmpty()) {
+            System.out.println("Following intermediate topics are not found, skipping them");
+            for (final String topic : notFoundIntermediateTopics) {
+                System.out.println("Topic:" + topic);
+            }
+            topicNotFound = EXIT_CODE_ERROR;
```

Comment:
This was not in the original patch, but seems to be expected by the fixed test case.

Reply:
Ack.